Usiqee/Module/Auth/PreAuthVC.swift

```diff
@@ -93,3 +182,56 @@ extension PreAuthVC: SignUpVCDelegate {
         delegate?.didSignIn()
     }
 }
+
+// MARK: - ASAuthorizationControllerDelegate
+@available(iOS 13.0, *)
+extension PreAuthVC: ASAuthorizationControllerDelegate {
+    
+    func authorizationController(controller: ASAuthorizationController, didCompleteWithAuthorization authorization: ASAuthorization) {
+        if let appleIDCredential = authorization.credential as? ASAuthorizationAppleIDCredential {
+            guard let nonce = currentNonce else {
+                fatalError("Invalid state: A login callback was received, but no login request was sent.")
```

Comment:
`fatalError` provoke a crash. we can display an error message maybe

Reply:
merge in one error message